OneCommunityGlobal/HGNRest
# Luis unit test permission change logs controller

# Description
Unit testing for Permission Change Logs controller



## Related PRS (if any):
N/A

## Main changes explained:
unit tests for the warnings controller.
Created tests for getPermissionChangeLogs
Created a md file describing all positive and negative cases. 

## How to test:
1. check into current branch
2. do` npm install`
3. run `npm run test permissionChangeLogsController`
4. verify all tests pass

## Screenshots or videos of changes:
<img width="1219" alt="Screenshot 2024-04-29 at 9 11 57 PM" src="https://github.com/OneCommunityGlobal/HGNRest/assets/22454953/7a6a2703-81be-4f13-885d-4a34d70b7ab6">


## Note:
N/A
